app/views/partials/theme/simple/dashboard/orders/index.liquid

```diff
@@ -1,60 +1,85 @@
 {% if orders.total_entries > 0 %}
-  <table class="table w-full table-auto">
-    <thead>
-      <tr>
-        <th class="px-4 py-2 text-lg font-medium text-left">{{ "app.created_at" | t }}</th>
-        <th class="px-4 py-2 text-lg font-medium text-left">{{ "app.models.orders.form.id" | t }}</th>
+  <section class="w-full lg:table">
+  
+    <header class="hidden lg:table-row font-bold">
+      <div class="lg:pl-2 lg:py-4 lg:table-cell">{{ 'app.created_at' | t }}</div>
+      <div class="lg:pl-2 lg:py-4 lg:table-cell">{{ 'app.models.orders.form.id' | t }}</div>
+      {% if role == 'buyer' %}
+        <div class="lg:pl-2 lg:py-4 lg:table-cell">{{ 'app.seller' | t }}</div>
+      {% endif %}
+      {% if role == 'seller' %}
+        <div class="lg:pl-2 lg:py-4 lg:table-cell">{{ 'app.buyer' | t }}</div>
+      {% endif %}
```

Comment:
```suggestion
      <div class="lg:pl-2 lg:py-4 lg:table-cell">{{ 'app.' | append: role | t }}</div>
```

Liquid is a kind of functional language

Reply:
This does not seem to solve the problem as it still needs to be in a condition and show only if the role is set to buyer or seller. I can go with this eventually:

`
{% if role == 'buyer' or role == 'seller' %}
`

But can you use this kind of appending to print what's in object in the next line?

`
{{ order.buyer.name }}
{{ order.seller.name }}
`